views/play.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Welcome <%= @name %> </h1>
+
+<form action="/play" method="post">
+  <input type="submit" name="weapon" value="Rock">
+  <input type="submit" name="weapon" value="Paper">
+  <input type="submit" name="weapon" value="Scissors">
```

Comment:
Use of a drop-down list for this form might have been easier than three individual buttons